### PR TITLE
Chore merge from 8fddb9f to a3f8480

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/8fddb9fe57ba4b9ee83aa35ec59cbe761c1e64d0
- [x] https://github.com/twbs/bootstrap/commit/f2568746756384ea3d1adc2ef6ecdf815c7796ef
- ~~https://github.com/twbs/bootstrap/commit/2dc329dd3725d99f756b07a70db5b1cd7432bcea~~: nothing to do here since the variants displayed in Bootstrap (in the buttons, breadcrumbs, badges, and jumbotrons examples) don't exist in our design guidelines. Designers have been contacted to check if some things could be integrated into our design guidelines or if already existing stuff is matching these examples. Let's tackle it in other PRs if needed.
- [x] https://github.com/twbs/bootstrap/commit/d86c316a88e8123c4cc8aaa17a0902557ee0df7a
- [x] https://github.com/twbs/bootstrap/commit/f8fa05d27e9766f479fc98a5be40fb3a19e62d7a
- ~~https://github.com/twbs/bootstrap/commit/2b31a6b0d5e8bc4b5cc6516ffe51f5937fffe3eb~~: nothing to do here, this fix is already in Boosted
- [x] https://github.com/twbs/bootstrap/commit/42e99cc59f6e87795e25d751bcfaf3be711191b4
  - ℹ️ I don't know what's really been applied by the authors so I stuck with my first approach upstream; be resizing images in https://www.iloveimg.com/ with a fixed width of 1000px to reduce the size. 
- [x] https://github.com/twbs/bootstrap/commit/31221272e0bbea1e436699cd5d68f705f37d3361
- ~~https://github.com/twbs/bootstrap/commit/5241b988c0156dbcc2c6bdaeb29967141bea567d~~: nothing to do since we don't have any dashboard examples in Boosted 
- [x] https://github.com/twbs/bootstrap/commit/d5765906321cc66a89123a3c4273695a4dbf9b5c
- [x] https://github.com/twbs/bootstrap/commit/28d9c0ccdd7d104192243815e5a75027044dd146
- [x] https://github.com/twbs/bootstrap/commit/a6d774c93e4667e8a426a9a4f6644bc1b0bebad8
- [x] https://github.com/twbs/bootstrap/commit/f44feabe16f4a0751428e486da2fd8396a3a16f2
- [x] https://github.com/twbs/bootstrap/commit/0d27f3c284f1b2450e6aee5c61e7040a30c8e190
- [x] https://github.com/twbs/bootstrap/commit/78f879eb57e56fc708f1e23d3643f8cecb813227
- [x] https://github.com/twbs/bootstrap/commit/afd37369bc87f8d48e5d86348a16dfbc9eba90f9
- ~~https://github.com/twbs/bootstrap/commit/2e2ca49633a11dc2a8281dfcd2952e0824f592b4~~ + ~~https://github.com/twbs/bootstrap/commit/c6345285c161e5760411d732a123f5a2f0a49b18~~: nothing to do here since already backported in Boosted
- [x] https://github.com/twbs/bootstrap/commit/51299bf4e125c0222412d3a3159cf2f0ccb957f5
- [x] https://github.com/twbs/bootstrap/commit/6b4b77e32052c28b9c338e813d4b44dc893c922d
- [x] https://github.com/twbs/bootstrap/commit/80ba84844399bd3e83ed77d42202ea8e7a3620d4
- [x] https://github.com/twbs/bootstrap/commit/23d20fa54f56df357501498719a5c63372bc49d0
- ~~https://github.com/twbs/bootstrap/commit/8a725016fe0978244c5cd45ce4bdb19b1b91e669~~: already done via Dependabot
- [x] https://github.com/twbs/bootstrap/commit/34901f1bc734cf4dd4eaa2e9581aeb72c3f9316e
- [x] https://github.com/twbs/bootstrap/commit/38a431068f2a05a8a76846f027c930e04f1c6c2b
- [x] https://github.com/twbs/bootstrap/commit/b8b81669f0c62d30bbaba405e31a6eec2f035421
- ~~https://github.com/twbs/bootstrap/commit/912cfe5823cea6f3ce133898a315f1a7611537f6~~: nothing to do since this example doesn't exist as is in Boosted
- [x] https://github.com/twbs/bootstrap/commit/783213691a5b90d4902fd6d35dd498a0e121eec8
- [x] https://github.com/twbs/bootstrap/commit/dbc021b220d29a3c87f6968fbc84860be311610f
- [x] https://github.com/twbs/bootstrap/commit/9a1f4ed9e5cba6424247928b786858edd38223da
- [x] https://github.com/twbs/bootstrap/commit/b26233c1ab9ffd2f927d908c7877b44766af3012
- [x] https://github.com/twbs/bootstrap/commit/ea19c3da8ac33c7676debfc2ac91c196e861f358    
- [x] https://github.com/twbs/bootstrap/commit/d5f4532b3eb0c1ae2c904bb58c488bc3da44b8f3
  - ⚠️ Sorry but I haven't had the time to check everything properly. Please double-check it 🙏   
- ~~https://github.com/twbs/bootstrap/commit/cff5c60a4507974be6cb6614c66377d30a2397be~~: nothing to do here since `.mt-1` would be worse in Boosted doc
- ~~https://github.com/twbs/bootstrap/commit/833da72712fbc1bb61807218acb60476dfc9e0ed~~: nothing to do since already fixed in Boosted
- [x] https://github.com/twbs/bootstrap/commit/423f112b2332e04d900a2668d6c0ac6c0c2e42ad
- [x] https://github.com/twbs/bootstrap/commit/039f3d1c8bd1653d1fde5d14f5194e30e33c3c80
- ~~https://github.com/twbs/bootstrap/commit/3cec38c368346594aefad5d6185c599c670b1c02~~: nothing to do since we don't have this headers example in Boosted
- [x] https://github.com/twbs/bootstrap/commit/a3f8480d5b5bb4fbb6be9f2f87b808648b4f0be4

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Funny that we remove 5kb from our images it lighten 80% of the size, Bootstrap removes 60kb per image and its only 20% 🤔 
One things to tackle and we're good to go !

File: scss/mixins/_buttons.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
   --#{$prefix}btn-hover-color: #{$hover-color};
   --#{$prefix}btn-hover-bg: #{$hover-background};
   --#{$prefix}btn-hover-border-color: #{$hover-border};
-  --#{$prefix}btn-focus-shadow-rgb: #{$btn-focus-shadow-color}; // Boosted mod
+  --#{$prefix}btn-focus-shadow-rgb: #{$btn-focus-shadow-color}; // Boosted mod: instead of `#{to-rgb(mix(color-contrast($link-color), $link-color, 15%))}`
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/mixins/_buttons.scss:30]: Oh wow, I'm gonna check. Thx LM

[the remark this answers is not in the document]
Comment (julien-deramond) [reply scss/mixins/_buttons.scss]: Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1882/commits/86cedf6e32033e0a981020ed6e2931559b84e29a

[the remark this answers is not in the document]
Comment (louismaximepiton) [reply scss/mixins/_buttons.scss]: Sounds good !

**Pull request merged**